doc/config.rst

```diff
@@ -800,6 +797,9 @@ that use Sphinx's HTMLWriter class.
    theme.  If you only want to add or override a few things compared to the
    theme's stylesheet, use CSS ``@import`` to import the theme's stylesheet.
 
+   .. deprecated:: 1.8
+      Use :confval:`html_stylesheets` instead.
```

Comment:
I noticed `html_style` can't replace this. I'll remove this from this PR. Let's discuss in another PR.